File: sorting/bubblesort/python/bubbleSort.py
#bubble sort

def Sort(input: list) -> list:
    for k in range(len(input)-1):
        for i in range(len(input)-k-1):
            if input[i]>input[i+1]:
                temp = input[i+1]
                input[i+1] = input[i]
                input[i] = temp
    return input

File: sorting/bubblesort/javascript/bubbleSort.js


function Sort(input)
{
  for(let k=0;k<input.length-1;k++)
  {
    for(let i=0;i<input.length-k-1;i++)
    {
      if(input[i]>input[i+1])
      {
        let temp = input[i];
        input[i] = input[i+1];
        input[i+1] = temp;
      }
    }
  }
  return input;
}


module.exports = {Sort};
